src/main/java/bio/terra/app/configuration/SamConfiguration.java

```diff
@@ -1,81 +1,25 @@
 package bio.terra.app.configuration;
 
 import org.springframework.boot.context.properties.ConfigurationProperties;
-import org.springframework.boot.context.properties.EnableConfigurationProperties;
-import org.springframework.context.annotation.Configuration;
+import org.springframework.boot.context.properties.ConstructorBinding;
 
-@Configuration
-@EnableConfigurationProperties
-@ConfigurationProperties(prefix = "sam")
-public class SamConfiguration {
-  private String basePath;
-  private String stewardsGroupEmail;
-  private String adminsGroupEmail;
-  private int retryInitialWaitSeconds;
-  private int retryMaximumWaitSeconds;
-  private int operationTimeoutSeconds;
-
-  public String getBasePath() {
-    return basePath;
-  }
-
-  public void setBasePath(String basePath) {
-    this.basePath = basePath;
-  }
-
-  public String getStewardsGroupEmail() {
-    return stewardsGroupEmail;
-  }
-
-  public void setStewardsGroupEmail(String stewardsGroupEmail) {
-    this.stewardsGroupEmail = stewardsGroupEmail;
-  }
-
-  public String getAdminsGroupEmail() {
-    return adminsGroupEmail;
-  }
-
-  public void setAdminsGroupEmail(String adminsGroupEmail) {
-    this.adminsGroupEmail = adminsGroupEmail;
-  }
+/* SAM Retry notes:
```

Comment:
I think it's OK to leave this as a `//` comment. If you want to convert it to a `/* */` comment, then it should look like
```
/*
 * First paragraph...
 * 
 * Second paragraph...
 */
```
https://google.github.io/styleguide/javaguide.html#s4.8.6-comments